client/src/components/SaladListComponent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react"
+import SaladTile from "./SaladTile"
+
+
+const SaladListComponent = (props) => {
```

Comment:
should only have one empty line separating - can remove either line 3 or 4
https://github.com/airbnb/javascript#whitespace--no-multiple-blanks